### PR TITLE
Fix performance when pasting thousands of values.

### PR DESCRIPTION
**Pull request by rlm1023**
Fixes an issue where pasting many values was slow due to https://github.com/looker/ui-select/pull/16

### CODE REVIEW

File: src/uiSelectMultipleDirective.js
Comment (rlm1023) [inline src/uiSelectMultipleDirective.js:219]: I think this broke some things without the onSelectCallback stuff above. Hold off for a second while I work on fixing it.

**Review by lameyer — COMMENTED** [review]
I know you're not done, but it would be great to have a test for this! I can show you how the tests work if you have any Qs!

File: src/uiSelectMultipleDirective.js
Comment (rlm1023) [reply src/uiSelectMultipleDirective.js:219]: On closer examination I don't think I broke anything. The onSelectCallback is only used in a few instances and no where that we can actually paste into. It could be an enhancement in the future, but for now I think this is fine.

**Comment by rlm1023** [comment]
You mean a test for pasting? or pasting a ton of values?

**Comment by lameyer** [comment]
A test to make sure that we don't do a bunch of work when pasting and instead follow this code path I think would be good. I have some pasting tests in `test/select.spec.js` and you probably could just modify the tests I wrote in https://github.com/looker/ui-select/pull/16

Lemme know if you have an Qs tho!

**Comment by rlm1023** [comment]
Here's a plnkr for it http://run.plnkr.co/plunks/kdyMAHbsuelPQboJ1eBI/

I **don't** recommend pasting large values into plnkr as it doesn't appear to be able to handle it.

File: test/select.spec.js
Comment (lameyer) [inline test/select.spec.js:2921]: Probably want to update this test too to make sure it doesn't broadcast `uis:select-multiple` instead here:

https://github.com/looker/ui-select/blob/9b816c11cc117219e47756630af5ee9cddf0206e/test/select.spec.js#L2929

**Review by lameyer — APPROVED** [review]
LGTM! Confirmed this solves the performance issues.

I just had one comment about updating one other test, but once that's updated, merge away! I assume you ran all the tests locally and they passed 👍

File: test/select.spec.js
Comment (rlm1023) [reply test/select.spec.js:2921]: Oh yeah, you're right. Otherwise this will sneakily pass.

**Pull request merged**
